src/ux/chooseDpath/ChooseDPath.tsx

```diff
@@ -72,9 +74,11 @@ export const ChooseDPathComponent: React.FC<Interface> = ({
     setIsLoading(true)
     setViewAbleAccounts([])
 
-    const currentIndexes = Array.from({ length: 5 }, (_, i) => i + startingIndex)
+    // if the provider's path is set to Ethereum, use that, else use the ChainIds:
+    const chainId = (provider.path === ETHEREUM_DPATH || provider.dpath === ETHEREUM_DPATH) ? 1 : provider.chainId
+    const nextPaths = [0, 1, 2, 3, 4].map(int => getDPathByChainId(chainId, int))
```

Comment:
Should we do something like

```suggestion
    const baseDpath = (provider.path || provider.dpath).split('/').slice(0, -1).join('/')
    const nextPaths = [0, 1, 2, 3, 4].map(index => `${baseDpath}/${index}`)
```

Reply:
And, for another PR, should we fix this `provider.path || provider.dpath` in `providers`?

Reply:
For the suggested change, I would instead suggest that in our `@rsksmart/rlogin-dpath` we have a function called getNextPath. That would be better readability for the next developer.

For changing .path and .dpath, yes, I created this ticket here: https://github.com/rsksmart/rLogin-providers/issues/47